pyinter/interval.py

```diff
@@ -195,6 +195,14 @@ def _get_new_lower_upper(self, other, operator):
             else:
                 new_upper = self._upper
         return new_lower, new_upper
+    
+    def is_empty(self):
+        """
+        Returns True iff there are no elements inside this interval.
+        """
+        return (self._lower_value >= self._upper_value and
+                self._lower == self.OPEN and
+                self._upper == self.OPEN)
```

Comment:
What is both `_upper` and `_lower` are closed but `_lower` is greater than `_upper`?